rb_ws/src/buggy/scripts/auton/model_predictive_controller.py

```diff
@@ -44,28 +47,25 @@ class ModelPredictiveController(Controller):
     N_CONTROLS = 1
 
     # MPC Cost Weights
-    state_cost = np.array([0.0001, 250, 5, 25])  # x, y, theta, steer
+    state_cost = np.array([0.0001, 250, 5, 2500])  # x, y, theta, steer
     control_cost = np.array([5])  # d_steer
     final_state_cost = 2 * np.array([0.0001, 250, 5, 25])  # x, y, theta, steer
 
-    # State constraints (relative to the reference)
-    state_lb = np.array([-np.inf, -np.inf, -np.inf, -np.pi / 9])  # x, y, theta, steer
-    state_ub = np.array([np.inf, np.inf, np.inf, np.pi / 9])  # x, y, theta, steer
-
     # Control constraints
-    control_lb = np.array([-np.pi * 2])  # d_steer
-    control_ub = np.array([np.pi * 2])  # d_steer
-    # control_lb = np.array([-np.inf])  # d_steer
-    # control_ub = np.array([np.inf])  # d_steer
+    # control_lb = np.array([-np.pi / 2])  # d_steer
+    # control_ub = np.array([np.pi / 2])  # d_steer
+    control_lb = np.array([-np.inf])  # d_steer
+    control_ub = np.array([np.inf])  # d_steer
```

Comment:
Is this what removes any steering maxes?

Reply:
this removes steering rate maxes, not steering maxes